docs/changelog.rst

```diff
@@ -3,4 +3,4 @@
 Changelog
 =========
 
-.. include:: ../CHANGELOG.rst
+.. include:: ../CHANGELOG.md
```

Comment:
@mmetak I'm not sure to be honest, when @cdrage builds the changelog it's in md format, but we use RST for the docs. We might have to see how much effort it would take to modify the format of the changelog. I would rather do that than update all the docs.